### DISCUSSION TITLE
Matrix field - add option to start with callapsed block

### DISCUSSION THREAD

**Discussion by piotrpog** [craftcms/cms#8803] [Ideas]
This might sound rather counterintuitive, but bear with me.

I often use matrix block to group some fields. Such matrix fields has min and max blocks setting set to `1`. And while such matrix field always starts with initial block already existing, sometimes i would rather this block be callapsed at first.

**Comment by bencresty** [comment]
+1 
Having lots of blocks on a page it's easier for us / clients to always start with all blocks collapsed, so we can open only the one we wish to edit (we prefer this to be optionally per page / entry tho!)
